usagov/usagov-2021
# USAGOV-1533: Check whether tome is running

When we check whether tome is running, can we check tome-static instead of tome-run?


https://cm-jira.usa.gov/browse/USAGOV-1533

## Description
When a user loads the page at https://localhost/static-site-status the script will now confirm whether Tome is disabled before reporting the Tome status. In the event the flag that indicates that Tome is still running yet Tome is disabled, the flag gets reset.

## Type of Changes


## Testing Instructions
* Pull in the latest database
* set Tome to be disabled
* go to https://localhost/static-site-status
* Expect to see a message saying that Tome is not allowed to run
* Additionally if you start Tome, kill it (before it can report that it is done running), disable it, and then go to https://localhost/static-site-status again. The script is smart enough to reset the flag that indicates tome is running.

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
